src/store/tokens.ts

```diff
@@ -120,20 +129,36 @@ export const prepareAndExecuteTokenTransactions = async (
 };
 
 export const prepareAndExecuteTransactions = async (operations: Transaction[] = []) => {
-  const { account, logicContract } = await getBurrow();
+  const { account, logicContract, view } = await getBurrow();
   const transactions: Transaction[] = [];
 
+  const storageDepositTransaction = {
+    receiverId: logicContract.contractId,
+    functionCalls: [
+      {
+        methodName: ChangeMethodsLogic[ChangeMethodsLogic.storage_deposit],
+        attachedDeposit: new BN(expandToken(NEAR_STORAGE_DEPOSIT, NEAR_DECIMALS)),
+      },
+    ],
+  };
+
   // check if account is registered in burrow cash
   if (!(await isRegistered(account.accountId, logicContract))) {
-    transactions.push({
-      receiverId: logicContract.contractId,
-      functionCalls: [
-        {
-          methodName: ChangeMethodsLogic[ChangeMethodsLogic.storage_deposit],
-          attachedDeposit: new BN(expandToken(STORAGE_DEPOSIT_FEE, NEAR_DECIMALS)),
-        },
-      ],
-    });
+    transactions.push(storageDepositTransaction);
+  } else {
+    const balance = (await view(
+      logicContract,
+      ViewMethodsLogic[ViewMethodsLogic.storage_balance_of],
+      {
+        account_id: account.accountId,
+      },
+    )) as Balance;
+    const balanceTotalDecimal = new Decimal(balance.total);
```

Comment:
Let's use `balance.available` instead. This way we can manage used storage instead of the total storage.
Then we need to change `NEAR_STORAGE_DEPOSIT_MIN` to `0.05`.

But we should do 2 different deposits. The first deposit should be `0.15` as before, but the next deposit should be only `0.1`